src/xdocs/checks/naming/constantname.xml

```diff
@@ -69,15 +69,18 @@
       </subsection>
 
       <subsection name="Examples" id="Examples">
-        <p>
+        <p id="Example1-config">
```

Comment:
I added these ids in the templates. That's why they appear here. 

Please refer to https://github.com/checkstyle/checkstyle/issues/13205#issuecomment-1589523232. We want to be able to add anchors to the examples and it's easier for me if I add those anchors now.